### PR TITLE
12157: add project editor list to project page

### PR DESCRIPTION
**Pull request by trbmcginnis**
List of project editors displays on individual project page.

Fixes [AB#12157](https://dcp-paperless.visualstudio.com/d36fd830-9029-4b77-b0c4-b0df2012eb98/_workitems/edit/12157)

### CODE REVIEW

File: client/app/models/project-applicant.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
```
Comment (godfreyyeung) [inline client/app/models/project-applicant.js:1]: Ah, so in CRM, pas applicant entities are different from project applicant entities! CRM is weird.

File: server/src/projects/project-applicants/project-applicants.attrs.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const PROJECTAPPLICANT_ATTRS = [
+  'dcp_name',
+  'emailaddress',
+  'dcp_projectapplicantid',
+  'dcp_applicantrole',
+];
```
Comment (godfreyyeung) [inline server/src/projects/project-applicants/project-applicants.attrs.ts:6]: nit: line break here

**Comment by godfreyyeung** [comment]
Nice Taylor! Looks awesome!

File: server/src/projects/project-applicants/project-applicants.attrs.ts (hunk 2 above)
Comment (andycochran) [reply server/src/projects/project-applicants/project-applicants.attrs.ts:6]: We should all set up our text editors to trim trailing whitespace and ensure one trailing new line upon save. Atom's "whitespace" package does this and I often get lots of whitespace diffs when making small changes to files others have worked on.

**Pull request merged**